augur/merge.py

```diff
@@ -34,7 +38,21 @@
 future.  The SQLite 3 CLI, sqlite3, must be available.  If it's not on PATH (or
 you want to use a version different from what's on PATH), set the SQLITE3
 environment variable to path of the desired sqlite3 executable.
+
+**Sequences**
+
+Sequence files are merged in the order given. Naming files is optional. If
+names are provided, they will be checked against any named metadata for order
+and IDs.
+
+SeqKit is used behind the scenes to implement the merge, but, at least for now,
+this should be considered an implementation detail that may change in the
+future.  The CLI program seqkit must be available.  If it's not on PATH (or
+you want to use a version different from what's on PATH), set the SEQKIT
+environment variable to path of the desired seqkit executable.
```

Comment:
nitnitnit

```suggestion
future.  The CLI program seqkit must be available.  If it's not in $PATH (or
you want to use a version different from what's in $PATH), set the SEQKIT
environment variable to the full path of the desired seqkit executable.
```

Reply:
Victor's just following language I introduced; see the description of `SQLITE` in the same `--help` output.

The phrasing "on PATH" is fairly conventional. First, the environment variable name is just `PATH` not `$PATH` and second, the program isn't _in_ `PATH` itself, it's in one of the directories in `PATH`. This is conventionally described as being "on PATH".

Reply:
Public code on GitHub is not definitive, but "on PATH" is [widely used](https://github.com/search?q=%2F%28%3F-i%29%28on+PATH%7Con+your+PATH%7Con+the+PATH%29%2F&type=code) to describe a binary's location. The "in PATH" phrasing is also [widely used](https://github.com/search?q=%2F%28%3F-i%29%28in+PATH%7Cin+your+PATH%7Cin+the+PATH%29%2F&type=code), but much of that usage is referring to directories appearing in PATH not just binaries found in those directories.

We can totally use "in PATH" if you want, we should just be consistent.

Reply:
I'm fine with "on PATH" if that's the house style, no worries.